components/script/dom/mod.rs

```diff
@@ -415,11 +417,10 @@ pub mod performance;
 pub mod performanceentry;
 pub mod performancemark;
 pub mod performancemeasure;
-pub mod performancenavigationtiming;
```

Comment:
This PR is incorrectly reverting some changes that exist on master; I presume you encountered merge conflicts? Please correct the files so only your new changes are added.